changelogs/fragments/62587-module_utils-network-cloudengine.yml

```
@@ -0,0 +1,3 @@
+bugfixes:
+  - "Cloudengine module_utils - the ``set-id`` (RPC-REPLY XML attribute) may change over the time althougth ``set-id`` is the identity of the next RPC packet."
+  - "Cloudengine netconf plugin - add a dispatch RPC function,just return original RPC-REPLY, the function is used by ``Cloudengine module_utils``."
```

```suggestion
  - "Cloudengine netconf plugin - add a dispatch RPC function, just return original RPC-REPLY, the function is used by Cloudengine ``module_utils``."
```
Two tiny details. Besides this, I guess ready to merge if @hwDCN wants no further changes :)